kibbeh/src/ui/ProfileScheduled.tsx

```diff
@@ -0,0 +1,56 @@
+import { ScheduledRoom, UserWithFollowInfo, wrap } from "@dogehouse/kebab";
+import React, { useEffect, useState } from "react";
+import { useConn } from "../shared-hooks/useConn";
+import { useTypeSafeTranslation } from "../shared-hooks/useTypeSafeTranslation";
+import { ScheduledRoomCard } from "../modules/scheduled-rooms/ScheduledRoomCard";
+
+export interface ProfileScheduledProps
+  extends React.HTMLAttributes<HTMLDivElement> {
+  user: UserWithFollowInfo;
+}
+
+export const ProfileScheduled: React.FC<ProfileScheduledProps> = ({
+  user,
+  className = "",
+}) => {
+  const { t } = useTypeSafeTranslation();
+  const conn = useConn();
+  const wrapper = wrap(conn);
+
+  const [scheduledRooms, setScheduledRooms] = useState([]);
+  useEffect(() => {
+    wrapper.connection
+      .sendCall("room:get_scheduled", {
+        cursor: "",
+        range: "all",
+        userId: user.id,
+      })
+      .then((res: any) => {
+        setScheduledRooms(res.rooms);
+      });
+  }, [user.id]);
+
+  return (
+    <div
+      className={`mt-2 rounded-8 w-full leading-8 ${className}`}
+      style={{ maxWidth: 640 }}
+    >
+      {scheduledRooms &&
```

Comment:
you should be able to merge both (scheduledRooms & no rooms) logic blocks into one

Reply:
done